s3-pit-restore

```diff
@@ -58,7 +58,8 @@ class TestS3PitRestore(unittest.TestCase):
         if args.dest_prefix:
             base_path = os.path.join(args.dest_prefix, base_path)
 
-        s3 = boto3.resource('s3')
+        s3 = boto3.resource('s3', endpoint_url=(None or args.endpoint_url))
```

Comment:
It work because when args.endpoint_url is None, None or None gives None!

So that can be simplified as
```
 endpoint_url=args.endpoint_url
 ```